.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: Tests
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "tmp/distribution" ]
```

Comment:
Is "tmp/distribution" a leftover here?

Reply:
I am afraid we need that branch for building wheels, until we will change the flow to be relaying on release tags, but for that we need to spend more time.

Reply:
Will replace that condition with `tags` condition.

Reply:
I think we should address that before merging this into main.